charts/argo-cd/ci/ha-autoscaling-values.yaml

```diff
@@ -4,6 +4,8 @@ crds:
 
 redis-ha:
   enabled: true
+  exporter:
+    enabled: true
```

Comment:
In order to notice the ImagePullBackoff from next time 😇

Reply:
Maxbe we enable the exporter also for non-HA redis?
Shall we add this to the file `ci/default-values.yaml`?

Reply:
Oh you are right, I will add commit for this. 🙏

Reply:
I handled in https://github.com/argoproj/argo-helm/pull/3231/commits/4e7ca2bd314dc7054e84f0c438fe7661a67d45d4  🙋